README.md

```diff
@@ -1,5 +1,35 @@
 # Readium Desktop
 
+Import and read EPUB ebooks on your Windows 10/10S computer. Great reading experience; local bookshelf with search capabilities and personal
+tagging; highly customizable layout settings, easy navigation using the table of contents and page list, reading progress bar, labelled bookmarks, "Zen" mode.
+
+* Enjoy complex EPUB 3 ebooks, fixed layout, with audio, video, animation and interactivity.
+* Open Readium LCP protected ebooks provided by a Public Library or Bookseller
+* Access to free ebooks via an easy navigation into online catalogs (OPDS).
+* Test full keyboard navigation.
+* Vision impaired people: use NVDA or Narrator as a screen reader.
+
+No ads. No private data going anywhere.
+
+The application is currently localized in three languages: English, French and German. More languages will come soon.
```

Comment:
"More languages will come soon"
=>
I would replace this with something like "Language contributions are welcome", because I "will come soon" sounds very optimistic.